From d5f82ce972ab2f1d8714d7b166554d946b7ed24b Mon Sep 17 00:00:00 2001
From: Isuru Fernando <isuruf@gmail.com>
Date: Sun, 25 Aug 2024 20:51:53 -0500
Subject: [PATCH 2/2] use thread_local on windows

---
 src/flint.h.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/flint.h.in b/src/flint.h.in
index 0bafefb13..6d563033e 100644
--- a/src/flint.h.in
+++ b/src/flint.h.in
@@ -150,7 +150,7 @@ typedef struct __FLINT_FILE FLINT_FILE;
 #endif
 
 #if FLINT_USES_TLS
-# if __STDC_VERSION__ < 202300L && !defined(__cplusplus)
+# if 0
 #  define FLINT_TLS_PREFIX _Thread_local
 # else
 #  define FLINT_TLS_PREFIX thread_local
-- 
2.44.0

