From 88bcb2eee75c15e2f99c4634cbd4467c04fdb204 Mon Sep 17 00:00:00 2001
From: Fredrik Johansson <fredrik.johansson@gmail.com>
Date: Wed, 25 Oct 2023 10:34:37 +0200
Subject: [PATCH] CMake fix FLINT_VERSION_FULL

---
 CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 54f6aa9fa1..a36269d3f1 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -21,10 +21,10 @@ include(CheckLibraryExists)
 include(TestBigEndian)
 
 # Source of truth for project version
-file(STRINGS VERSION FLINT_VERSION)
+file(STRINGS VERSION FLINT_VERSION_FULL)
 
 project(flint
-  VERSION ${FLINT_VERSION}
+  VERSION ${FLINT_VERSION_FULL}
   DESCRIPTION "Fast Library for Number Theory"
   HOMEPAGE_URL https://flintlib.org/
   LANGUAGES C CXX)
